### PR TITLE
Update CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by shobhu98**
Some grammatical errors

### CODE REVIEW

**Comment by shobhu98** [comment]
Done!

**Comment by shobhu98** [comment]
Can you review the changes now?

**Comment by shobhu98** [comment]
Can you review the changes now?

File: .github/CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -3,9 +3,9 @@
 ## Be friendly and patient
 
 We understand that everyone has different levels of experience or knowledge in many diverse fields, be it technical or
-non-technical in nature. We also have areas of knowledge we are eager to expand; we want to be a community where people
+non-technical In nature. We also have areas of knowledge we are eager to expand; we want to be a community where people
```
Comment (craigtaub) [inline .github/CODE_OF_CONDUCT.md:6]: Why would `in` be uppercase?

Comment (shobhu98) [reply .github/CODE_OF_CONDUCT.md:6]: I am a BTech student with a minor in English. You can trust me. Yes I should be capital over here.

Comment (craigtaub) [reply .github/CODE_OF_CONDUCT.md:6]: > You can trust me

This is not at all how contributing to mocha works.

In order to have an effective code review it is necessary for the reviewer to have confidence in the change. I will require either proof or an explanation as to why each one is correct. That is not a lot to ask, for any code change you should be able to explain it.

Comment (shobhu98) [reply .github/CODE_OF_CONDUCT.md:6]: So I checked this on Grammarly, and it gives the same errors that I have corrected. Moreover Grammarly is showing some additional issues as well.
Hope this proof is justified.

**Comment by boneskull** [comment]
No thanks.  We need to look to the OJSF for guidance on changing the CoC.

**Pull request closed without merging**